frontend/apps/filemanager/filemanager.lua

```diff
@@ -128,6 +128,7 @@ function FileManager:setupLayout()
         right_icon_hold_callback = false, -- propagate long-press to dispatcher
     }
 
+    local show_finished = G_reader_settings:hasNot("show_finished") or G_reader_settings:isTrue("show_finished")
```

Comment:
Redudant, `hasNot` will never be true if `isTrue` is, so this can be reduced to `isTrue`.

Reply:
Wait. Brainfart ;p.

`nilOrTrue` is what you want ;).

Reply:
I wanted that setting to be *true* by default, which `isTrue()` does not grant. I couldn't find a way to declare a default settings value. My experience is that a plain `isTrue()` will return false if the value is not set (nil), while `hasNot()` returns true.
If `hasNot() or isTrue()` isn't suitable there, what is the idiomatic way with koreader to check for *nil or true*?

Reply:
Don't we have a helper function called nilOrTrue()? :-P

Reply:
Yep, see my followup ;).  (GH's weird update notifications strike again, I guess ;)).

Reply:
The second comment didn't show up in my browser yesterday, so sorry my reply was useless. Thanks for the advice.